swagger_server/controllers/submitter/encrypter.py

```diff
@@ -0,0 +1,78 @@
+import os
+
+import jwt
+from cryptography.hazmat.backends.openssl.backend import backend
+from cryptography.hazmat.primitives import hashes
+from cryptography.hazmat.primitives import serialization
+from cryptography.hazmat.primitives.asymmetric import padding
+from cryptography.hazmat.primitives.ciphers import Cipher
+from cryptography.hazmat.primitives.ciphers import algorithms
+from cryptography.hazmat.primitives.ciphers import modes
+
+from swagger_server.controllers.cryptography.jwe_encryption import JWEEncrypter
+from swagger_server.controllers.helper import to_bytes, to_str
+from swagger_server.controllers.cryptography.keys import TEST_PRIVATE_SIGNING_KEY, TEST_PRIVATE_SIGNING_KEY_PASSWORD, \
+    TEST_SDX_PUBLIC_KEY
+KID = 'EDCSR'
+
+
+class Encrypter(JWEEncrypter):
+
+    # password and key variables are dynamically assigned
+    def __init__(self):
+
+        private_key = os.environ.get('PRIVATE_SIGNING_KEY', TEST_PRIVATE_SIGNING_KEY)
+        private_key_password = os.environ.get('PRIVATE_SIGNING_KEY_PASSWORD', TEST_PRIVATE_SIGNING_KEY_PASSWORD)
+        public_key = os.environ.get('SDX_PUBLIC_KEY', TEST_SDX_PUBLIC_KEY)
+
+        self._load_keys(private_key, private_key_password, public_key)
+
+        # first generate a random key
+        self.cek = os.urandom(32)  # 256 bit random CEK
```

Comment:
I'm guessing this is monkey-patched in the unit tests? Might be somewhere else to consider DI.